src/main/java/com/powsybl/openloadflow/sa/extensions/ContingencyLoadFlowParametersJsonSerializer.java

```diff
@@ -0,0 +1,60 @@
+package com.powsybl.openloadflow.sa.extensions;
```

Comment:
Add copyright

Reply:
fixed in [74c7960](https://github.com/powsybl/powsybl-open-loadflow/pull/1139/commits/74c79605b4411a6e1ddb50aa9f2a6476625fc91a)